static/elements/chromedash-legend.html

```diff
@@ -42,11 +42,11 @@
     <section>
       <label>Example search queries</label>
       <ul class="queries">
-        <li><span>"shipped_milestone&lt;30"</span>features that landed before 30</li>
-        <li><span>"shipped_android_milestone&gt;40"</span>features that landed on Anrdoid after 40 (milestone types: android, ios, webview, opera, opera_android)</li>
-        <li><span>"shipped_milestone&lt;=50"</span>features in desktop chrome 50</li>
+        <li><span>"browsers.chrome.desktop&lt;30"</span>features that landed before 30</li>
```

Comment:
Agreed. From the UX perspective, requiring folks to map our data model to search queries is friendly. I would be for supporting a more friendly shorthand. Would you be interested in helping with that after this PR lands?